docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+We will add our address as an Admin so we can add new Minter and Admin addresses in the future. 
+
+**Use your own address** instead of `0x0000000b9af48743ef1188f3F20c9b8B90F52a5b`.
+
+```json
+{
+  "config": {
+    "contractNativeMinterConfig": {
+      "blockTimestamp": 0,
+      "adminAddresses": ["0x0000000b9af48743ef1188f3F20c9b8B90F52a5b"]
+    }
+  }
+}
+```
+
+## Restricting Contract Deployers
+
+We will deploy the contracts and players will use those contracts to play the game. So we don't need other people to deploy the contracts. We can restrict the smart contract deployers by using [`contractDeployerAllowListConfig`](#contractdeployerallowlistconfig).
+
+We will add our address as an Admin so we can add new Deployer and Admin addresses in the future. 
+
+**Use your own address** instead of `0x0000000b9af48743ef1188f3F20c9b8B90F52a5b`.
+
+```json
+{
+  "config": {
+    "contractDeployerAllowListConfig": {
+      "blockTimestamp": 0,
+      "adminAddresses": ["0x0000000b9af48743ef1188f3F20c9b8B90F52a5b"]
+    }
+  }
+}
+```
+
+Now we're done! The final config looks like:
+
+```json
+{
+    "config": {
+        "chainId": 321123,
+        "homesteadBlock": 0,
+        "eip150Block": 0,
+        "eip150Hash": "0x2086799aeebeae135c246c65021c82b4e15a2c451340993aacfd2751886514f0",
+        "eip155Block": 0,
+        "eip158Block": 0,
+        "byzantiumBlock": 0,
+        "constantinopleBlock": 0,
+        "petersburgBlock": 0,
+        "istanbulBlock": 0,
+        "muirGlacierBlock": 0,
+        "SubnetEVMTimestamp": 0,
+        "feeConfig": {
+            "gasLimit": 5000000,
+            "targetBlockRate": 5,
+            "minBaseFee": 60000000000,
+            "targetGas": 10000000,
+            "baseFeeChangeDenominator": 50,
+            "minBlockGasCost": 0,
+            "maxBlockGasCost": 5000000,
+            "blockGasCostStep": 10000
+        },
+        "contractDeployerAllowListConfig": {
+            "blockTimestamp": 0,
+            "adminAddresses": [
+                "0x0000000b9af48743ef1188f3F20c9b8B90F52a5b"
+            ]
+        },
+        "contractNativeMinterConfig": {
+            "blockTimestamp": 0,
+            "adminAddresses": [
+                "0x0000000b9af48743ef1188f3F20c9b8B90F52a5b"
+            ]
+        }
+    },
+    "alloc": {
+        "0000000b9af48743ef1188f3F20c9b8B90F52a5b": {
+            "balance": "0x3635C9ADC5DEA00000"
+        }
+    },
+    "nonce": "0x0",
+    "timestamp": "0x0",
+    "extraData": "0x00",
+    "gasLimit": "5000000",
+    "difficulty": "0x0",
+    "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "number": "0x0",
+    "gasUsed": "0x0",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+# Creating and Deploying the Subnet Using the Custom Genesis File
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) You shouldn't have multiple `<h1>`s in the same document.

> MD025/single-title/single-h1: Multiple top-level headings in the same document

[More info: MD025](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document)